frontend/integration-tests/tests/kubevirt/vm.wizard.scenario.ts

```diff
@@ -45,30 +67,58 @@ describe('Kubevirt create VM using wizard', () => {
       },
       networkResources: [networkInterface],
       storageResources: [rootDisk],
+    })
+    .set('ClonedDisk', {
+      provision: {
+        method: 'Cloned Disk',
+      },
+      networkResources: [networkInterface],
+      storageResources: [diskToCloneFrom],
     });
 
   beforeAll(async() => {
-    createResource(multusNad);
+    createResources([multusNad, testDataVolume]);
   });
 
   afterAll(async() => {
-    deleteResource(multusNad);
+    deleteResources([multusNad, testDataVolume]);
+  });
+
+  afterEach(() => {
     removeLeakedResources(leakedResources);
   });
 
   provisionConfigs.forEach((provisionConfig, configName) => {
     it(`Create VM using ${configName}.`, async() => {
-      const vmConfig = {
-        ...commonSettings,
-        name: `vm-${provisionConfig.provision.method.toLowerCase()}-${testName}`,
-        provisionSource: provisionConfig.provision,
-        storageResources: provisionConfig.storageResources,
-        networkResources: provisionConfig.networkResources,
-      };
-      const vm = new VirtualMachine(vmConfig);
+      const vm = new VirtualMachine(vmConfig(configName.toLowerCase(), provisionConfig));
       await withResource(leakedResources, vm.asResource(), async() => {
-        await vm.create(vmConfig);
+        await vm.create(vmConfig(configName.toLowerCase(), provisionConfig));
       });
     }, VM_BOOTUP_TIMEOUT);
   });
+
+  it('Multiple VMs created using "Cloned Disk" method from single source', async() => {
+    const clonedDiskProvisionConfig = provisionConfigs.get('ClonedDisk');
+    const vm1Config = vmConfig('vm1', clonedDiskProvisionConfig);
+    const vm2Config = vmConfig('vm2', clonedDiskProvisionConfig);
+    vm1Config.startOnCreation = false;
+    vm1Config.networkResources = [];
+    const vm1 = new VirtualMachine(vm1Config);
+    const vm2 = new VirtualMachine(vm2Config);
+
+    await withResource(leakedResources, vm1.asResource(), async() => {
+      await vm1.create(vm1Config);
+      // Don't wait for the first VM to be running
+      await vm1.action('Start', false);
+      await withResource(leakedResources, vm2.asResource(), async() => {
+        await vm2.create(vm2Config);
+        await vm1.waitForStatusIcon(statusIcons.running, VM_BOOTUP_TIMEOUT);
+
+        // Verify that DV of VM created with Cloned disk method points to correct PVC
+        const dvResource = getResourceObject(`${vm1.name}-${testDataVolume.metadata.name}-clone`, vm1.namespace, 'dv');
+        const pvcSource = _.get(dvResource, 'spec.source.pvc', {});
+        expect(pvcSource).toEqual({name: testDataVolume.metadata.name, namespace: testDataVolume.metadata.namespace});
```

Comment:
Am I correct that we are trying to clone both VMs? Shouldn't we also check the 2nd VM here?

Reply:
Or what is the reasoning behind having two VMs in this TC?

Reply:
Tomas suggested to use one dataVolume for creating 2 VMs to ensure multiple VMs can be creted from single source.
So I decided to verify content of the first one since both VMs are created in the same way.

Reply:
ok